src/test/java/core/basesyntax/service/impl/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.ReaderService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReaderServiceImplTest {
+    private static ReaderService readerService;
+    private static final String EMPTY_FILE_PATH = "src/test/resources/emptyFile.csv";
+    private static final String CORRECT_FILE_PATH = "src/test/resources/okDataFile.csv";
+    private static final String NON_EXISTENT_FILE_PATH = "src/test/resources/nonExistent.csv";
+
+    @BeforeAll
+    static void beforeAll() {
+        readerService = new ReaderServiceImpl();
+    }
+
+    @Test
+    void readCorrectDataFile_Ok() {
```

Comment:
```suggestion
    void read_CorrectDataFile_Ok() {
```
test method name should contain three parts separated by underscore